addons/auth_signup/models/res_users.py

```diff
@@ -221,8 +221,12 @@ def send_unregistered_user_reminder(self, after_days=5):
 
         # For sending mail to all the invitors about their invited users
         for user in invited_users:
-            template = self.env.ref('auth_signup.mail_template_data_unregistered_users').with_context(dbname=self._cr.dbname, invited_users=invited_users[user])
-            template.send_mail(user, email_layout_xmlid='mail.mail_notification_light', force_send=False)
+            email_template = self.env.ref('auth_signup.mail_template_data_unregistered_users', raise_if_not_found=False)
+            if email_template:
+                template = email_template.with_context(dbname=self._cr.dbname, invited_users=invited_users[user])
+                template.send_mail(user, email_layout_xmlid='mail.mail_notification_light', force_send=False)
+            else:
+                _logger.warning("Invalid template. It seems it has been deleted.")
```

Comment:
Maybe add more informations to the warning ?

For example
```suggestion
                _logger.warning('Template "auth_signup.mail_template_data_unregistered_users" was not found. Cannot send reminder notifications.')
```